Cargo.toml

```diff
@@ -37,7 +43,11 @@ derive_more = { git = "https://github.com/JelteF/derive_more", rev = "1196b2dd7a
 ] }
 
 serde = { version = "1.0.193", features = ["derive", "rc", "std"] }
-serde_json = { version = "1.0.108", features = ["preserve_order"] }
```

Comment:
pinning a few more dependencies, now only remains less than 5 not pinned, will work out during the coming weeks. Really need to get started on Profile Integration work...